tests/sparseml/transformers/finetune/finetune_oneshot_configs/gpu/gpu_config.yaml

```diff
@@ -0,0 +1,6 @@
+cadence: "weekly"
+test_type: "regression"
+model: "zoo:llama2-7b-llama2_pretrain-base"
+dataset: wikitext
+dataset_config_name: "wikitext-2-raw-v1"
```

Comment:
For the weekly tests I think we should use a model that has already been pretrained on a specific dataset then finetune on the same dataset (this is the typical research) flow. Maybe zoo:llama2-7b-ultrachat200k_llama2_pretrain-base and the ultrachat200k dataset